### PR TITLE
15772_fix - NR text change

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/15772

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2492-9hvuwveg.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2492-9hvuwveg.web.app

File: auth-web/src/components/auth/manage-business/AddNameRequestForm.vue [hunk 1]
```diff
@@ -288,6 +288,11 @@ export default class AddNameRequestForm extends Vue {
     margin-left: 0.5rem;
   }
 
+  .help-text {
+    justify-content: flex-start; // align help button to left
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/AddNameRequestForm.vue:292]: https://vuetifyjs.com/en/styles/flex/

**Review by seeker25 — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2492-9hvuwveg.web.app

**Pull request merged**